.ci/ci-setup.sh

```diff
@@ -19,8 +19,7 @@
 
 set -e -x
 
-root=$(cd `dirname "$0"/..`; pwd -P)
```

Comment:
Ah, it seems like the error was the end '`', it should have been:

```
root=$(cd `dirname "$0"`/..; pwd -P)
```

It may be useful to keep that to be able to invoke the script from any directory, not just the project root directory